lib/index.js

```diff
@@ -1,8 +1,11 @@
 const path = require('path');
 const BinWrapper = require('bin-wrapper');
+const pkgConf = require('pkg-conf');
 const pkg = require('../package');
 
-const hugoVersion = pkg.hugoVersion;
+const config = pkgConf.sync('hugo-bin');
+const hugoVersion = process.env.HUGO_BIN_VERSION || process.env.npm_config_hugo_bin_version || config.version || pkg.defaultHugoVersion;
```

Comment:
Are you sure `config.version` should be here?

Reply:
On second thought, supposing [checksum feature](https://github.com/fenneclab/hugo-bin/issues/36) for the future, it's not make sense to set hugoVersion on user configuration. 
Because it would force users to set all of version,os,platform,checksum in their configuraton file.

I remove this line.